### DISCUSSION TITLE
We found the patch versions not appeared in Golang Index.

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [minio/minio#17684] [Q&A] [answered]
Hello, we are a research team working on Golang. During our investigation, we discovered that the following CVEs - CVE-2023-28432, CVE-2023-28434, CVE-2023-28433, CVE-2023-27589, CVE-2023-25812, CVE-2022-35919, CVE-2022-31028, CVE-2022-24842, CVE-2021-43858, CVE-2021-41137, CVE-2021-21390, CVE-2021-21362, CVE-2021-21287, CVE-2020-11012 were addressed and the patch versions were released. However, we noticed that these patch versions have not appeared in the Golang Index, which means that 'go list' cannot automatically push the patch versions to downstream users.

We recommend that after releasing the versions, you push them to the Golang Index using the command 'go get github.com/minio/minio@version'. This will enable the automatic distribution of the patch versions to downstream users. Thank you for your attention.

**Comment by harshavardhana** [comment, marked as answer]
There were we way before Go did have vuln indexes. @Silence-worker-02 you can do `go get` yourself since it is anonymous. 

Along with that MinIO is a binary here not a library so not sure how they might show this to end users.

**Reply by Silence-worker-02** [reply]
Thank you for your attention and reply. You are right, the reason we asked this question was that, through the dependency relationships, we found there are 161 dependents using the vulnerable versions as a library.
